src/CoreWCF.Primitives/src/CoreWCF/IdentityModel/Selectors/SecurityTokenRequirement.cs

```diff
@@ -78,7 +78,7 @@ public bool RequireCryptographicToken
             }
         }
 
-        public SecurityKeyUsage KeyUsage
+        internal SecurityKeyUsage KeyUsage
         {
```

Comment:
Why make this internal? Same Q for KeyType

Reply:
SecurityKeyUsage is internal and SecurityKeyType too.. so either we can make those public or keep these internal.. I am more on the restrictive side

Reply:
We need to expose SecurityKeyType when we do WS-Federation support so it's best to expose that one now. SecurityKeyUsage doesn't need to be exposed.